package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "handlebars",
   "barename": "handlebars",
-  "version": "1.3.1-sugarcrm-temporary",
+  "version": "1.3.2-sugarcrm-temporary",
```

Comment:
Should we also update sidecar/package.json in this PR?

Reply:
I think that goes to the sidecar repo directly.